lms/urls.py

```diff
@@ -820,10 +820,16 @@
 
 if settings.FEATURES.get('ENABLE_OAUTH2_PROVIDER'):
     urlpatterns += (
+        # These URLs dispatch to django-oauth-toolkit or django-oauth2-provider as appropriate.
+        url(r'^oauth2/', include('lms.djangoapps.oauth_dispatch.urls')),
+        # These URLs contain the django-oauth2-provider default behavior (for views that have not
```

Comment:
If we create shims for the rest of the views in the dispatch app, then we wouldn't need to include the urls from the `edx_oauth2_provider`, right?  If so, is there a follow-up ticket to do so?